app/components/notification_component/notification_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="notification-component <%= @wrapper_css %>">
+  <div class="tooltip">
+    <%= image_pack_tag('media/images/icons/alert.svg') %>
+    <div class="tooltiptext">
+      <% @notifiable_programmes.each do |programme| %>
+        <p class="govuk-body-s">
+          <%= t('.text') %>
+        </p>
+        <%= link_to t('.link_text'), notification_link(programme), class: 'govuk-link' %>
```

Comment:
rather than having a helper, could there be a #notification_link method on the programme models?